test/e2e/tests/dapp-interactions/dapp-tx-edit.spec.js

```diff
@@ -68,29 +57,19 @@ describe('Editing confirmations of dapp initiated contract interactions', functi
         title: this.test.fullTitle(),
       },
       async ({ driver }) => {
-        await unlockWallet(driver);
+        await logInWithBalanceValidation(driver);
```

Comment:
same as above: we use logInWithBalanceValidation here since this test starts a transaction right away, so we need to make sure the balance is loaded to mitigate race conditions